uportal-war/src/main/webapp/WEB-INF/jsp/Marketplace/portlet/view.jsp

```diff
@@ -312,10 +310,10 @@
                                     <c:choose>
                                         <c:when test="${empty featuredPortlet.getParameter('mobileIconUrl')}">
 
-                                            <img src="${defaultIcon}">
+                                            <img src="${defaultIcon}" alt="">
```

Comment:
Icon for decoration only? Maybe ought to be `aria-hidden`? Possibly even at the containing `<div class="portlet-icon">` level?